WORKSPACE

```diff
@@ -102,6 +102,21 @@ git_repository(
     shallow_since = "1552938175 -0400",
 )
 
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+maven_install(
+    artifacts = [
+        # Test artifacts
+        "org.assertj:assertj-core:3.9.0",
+        "junit:junit:4.12"
+    ],
+    repositories = [
+        "https://jcenter.bintray.com/",
+        "https://maven.google.com",
+        "https://repo1.maven.org/maven2",
```

Comment:
pls prefer maven central to jcenter, basically put it in front jcenter :)

Reasons:

- Reliability
- Latency/Throughput
- Security (jcenter has had multiple instances of malicious artifacts uploaded to valid looking groupIds)